Source/DafnyCore/Compilers/SinglePassCompiler.cs

```diff
@@ -64,6 +66,23 @@ public abstract class SinglePassCompiler : Plugins.Compiler {
 
     public CoverageInstrumenter Coverage;
 
+    public virtual int RunProcess(ProcessStartInfo psi, Process process, String platform, TextWriter outputWriter) {
+      try {
+        if (process == null) {
+          process = Process.Start(psi);
+          if (process == null) {
+            return -1;
+          }
+        }
```

Comment:
```suggestion
        process ?= Process.Start(psi);
        if (process == null) {
          return -1;
        }
```

Reply:
Put in ??= -- if that is not what you actually meant, comment again.